lib/assets/package_application_patches/codesign_args_patch.diff

```diff
@@ -0,0 +1,33 @@
+From 13f2c76fd11b0d861ddf205d4d6ccc0c45a0d4b2 Mon Sep 17 00:00:00 2001
+From: Fabio Milano
+Date: Thu, 13 Aug 2015 15:02:41 +0200
+Subject: [PATCH] Fixed codesign_args
+
```

Comment:
I would recommend keeping a full description to the problem in the patch.

E.g. under https://gist.github.com/lacostej/f35f5fb8f3367551f22c#file-0001-packageapplication-still-passes-resource-rules-while-patch-L7

I used 
-------
Subject: [PATCH] PackageApplication still passes resource-rules while it is
 deprecated.

See https://developer.apple.com/library/mac/technotes/tn2206/_index.html#//apple_ref/doc/uid/DTS40007919-CH1-TNTAG206
-------

to make the patch self documented.